jupyterlab-ihaskell/src/index.ts

```diff
@@ -2,27 +2,22 @@ import {
   JupyterFrontEnd, JupyterFrontEndPlugin
 } from '@jupyterlab/application';
 
-import './codemirror-ihaskell';
+import { ICodeMirror } from '@jupyterlab/codemirror';
 
-import '../style/index.css';
+import { defineIHaskellMode } from './codemirror-ihaskell';
 
 /**
  * Initialization data for the extension1 extension.
  */
 const extension: JupyterFrontEndPlugin<void> = {
   id: 'ihaskell',
   autoStart: true,
-  requires: [],
-  activate: (app: JupyterFrontEnd) =>
+  requires: [ICodeMirror],
+  activate: (app: JupyterFrontEnd, codeMirror: ICodeMirror) =>
   {
-    app.serviceManager.ready
-      .then(() => {defineIHaskell()});
+    defineIHaskellMode(codeMirror);
+    console.log('ihaskell codemirror activated');
```

Comment:
hmmm... does it really need the log? i mean, if you've documented, "it works if you look in the console and see..." then keep it, but otherwise...

Reply:
Hmm, you're right that it's largely unnecessary. I can remove this.

Reply:
Fixed in https://github.com/gibiansky/IHaskell/commit/2638434cae3b5e9e12192a6ffdc60deedd0eb385.